src/nls/el/strings.js

```diff
@@ -120,11 +132,15 @@ define({
 
     "OPEN_FILE"                         : "Άνοιγμα Αρχείου",
     "SAVE_FILE_AS"                      : "Αποθήκευση Αρχείου",
-    "CHOOSE_FOLDER"                     : "Επίλεξε ένα φάκελο",
+    "CHOOSE_FOLDER"                     : "Επέλεξε ένα φάκελο",
```

Comment:
That's a tough one but it is "Επίλεξε". [See here](https://www.translatum.gr/forum/index.php?topic=173354.0) :)